metrics/influx/influx.go

```diff
@@ -123,10 +124,10 @@ func (in *Influx) WriteTo(w BatchPointsWriter) (err error) {
 	}
 
 	in.gauges.Reset().Walk(func(name string, lvs lv.LabelValues, values []float64) bool {
-		fields := fieldsFrom(lvs)
-		fields["value"] = last(values)
+		tags := mergeTags(in.tags, lvs)
 		var p *influxdb.Point
-		p, err = influxdb.NewPoint(name, in.tags, fields, now)
+		fields := map[string]interface{}{"value": last(values)}
+		p, err = influxdb.NewPoint(name, tags, fields, now)
```

Comment:
Can you please update the comment on the Influx struct to reflect the new role of fields and tags?